packages/grafana-ui/src/components/Flex/Flex.internal.story.tsx

```diff
@@ -40,166 +36,175 @@ const meta: Meta<typeof Flex> = {
 };
 
 export const Basic: StoryFn<typeof Flex> = ({ direction, wrap, alignItems, justifyContent, gap }) => {
+  const theme = useTheme2();
   return (
-    <Flex direction={direction} wrap={wrap} alignItems={alignItems} justifyContent={justifyContent} gap={gap}>
-      {Array.from({ length: 3 }).map((_, i) => (
-        <Item key={i} color="pink" text={i + 1} />
-      ))}
-    </Flex>
+    <div style={{ width: '300px', height: '300px', border: '1px solid grey' }}>
```

Comment:
The width of this should be bigger so we can play around with justifyContent.
Also annoyingly the height of the actual flex will not take up all the space of this box so it is hard to properly play around with this 😬  We would either need a `height` prop or a `grow` prop on the `Flex` for that to work... but we can think about this another time.